docs/ansible.netcommon.default_netconf.rst

```diff
@@ -1,4 +1,3 @@
-
 .. _ansible.netcommon.default_netconf:
```

Comment:
latest update to add_docs will remove the 'hint' part from the document.

Reply:
Hmmm, maybe it isn't triggering on non-modules

Reply:
yes , you are right.